ops/provision/modules/networking/security-groups.tf

```diff
@@ -12,6 +12,35 @@ resource "aws_security_group" "access" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
+
+  ingress {
+    cidr_blocks      = [
+      "18.205.248.83/32",
+    ]
+    description      = "AMS2 on digital-eks-dev cluster"
+    from_port        = 8080
+    ipv6_cidr_blocks = []
+    prefix_list_ids  = []
+    protocol         = "tcp"
+    security_groups  = []
+    self             = false
+    to_port          = 8080
+  }
+
+  ingress {
+    cidr_blocks      = [
+      "18.205.248.83/32",
+    ]
+    description      = "AMS2 on digital-eks-dev cluster"
+    from_port        = 8983
+    ipv6_cidr_blocks = []
+    prefix_list_ids  = []
+    protocol         = "tcp"
+    security_groups  = []
+    self             = false
+    to_port          = 8983
+  }
+
```

Comment:
☝️ these ingresses aren't needed if demo and prod are within `r2-gbh-ams` cluster, yeah?

Reply:
correct

Reply:
👍